### PR TITLE
Dubichtonya/textbook additional

### PR DESCRIPTION
**Pull request by DubichTonya**
Add styles and fix problems with settings and sounds.

### CODE REVIEW

**Review by Ars6300 — APPROVED** [review]

File: src/app/core/components/header/header.component.scss [hunk 1]
```diff
@@ -84,7 +84,7 @@ a:hover {
   background-size: 100% 2px;
 
   .page-header__icon {
-    background: rgba($color: var(--color-light), $alpha: 0.2);
+    background: rgba($color: #ffffff, $alpha: 0.2);
```
Comment (rss-maksim) [inline src/app/core/components/header/header.component.scss:87]: Why did you change to #ffffff? ```var(--color-light)``` doesn't work?

File: src/app/mini-games/sprint/utils/utils.ts [hunk 2]
```diff
@@ -39,6 +39,30 @@ export function playRawSound(src: string) {
   audio.src = src;
   audio.play();
 }
+export function playRawSoundArr(arr: string[]) {
+  let sequence_position = 0;
```
Comment (rss-maksim) [inline src/app/mini-games/sprint/utils/utils.ts:43]: We usually use camelCase naming variable

File: src/app/mini-games/sprint/utils/utils.ts [hunk 3]
```diff
@@ -39,6 +39,30 @@ export function playRawSound(src: string) {
...
+  function playSequence() {
+    const audio = new Audio();
+    audio.src = arr[sequence_position];
+    audio.removeEventListener('ended', prepareNextAudio);
+    audio.addEventListener('ended', prepareNextAudio, true);
+    audio.play();
+  }
+
+  function prepareNextAudio() {
+    sequence_position++;
```
Comment (rss-maksim) [inline src/app/mini-games/sprint/utils/utils.ts:60]: Let's use ```sequence_position += 1```

File: src/app/textbook/components/textbook-content/const.ts [hunk 4]
```diff
@@ -2,27 +2,27 @@ import { ITextbookContentTabs } from 'src/app/core/models/ITextbookContentTabs';
...
     group: '3',
   },
   {
-    title: 'Раздел 5',
+    title: '&#10029; &#10029; &#10029; &#10029; &#10029;',
     group: '4',
   },
   {
-    title: 'Раздел 6',
+    title: '&#10029; &#10029; &#10029; &#10029; &#10029; &#10029;',
```
Comment (rss-maksim) [inline src/app/textbook/components/textbook-content/const.ts:25]: You can rid of repetition by creating function, for example
```
generateStars(length: number): string {
  return ARray.from({ length }, () => '&#10029;').join(' ');
}
```

File: src/app/textbook/components/textbook-content/textbook-content.component.html [hunk 5]
```diff
@@ -5,15 +5,19 @@
   mat-raised-button
   >Словарь</a
 >
+<div class="textbook-content__info">
+  <span>Важная информация!</span> Разделы в учебнике поделены в зависимости от сложности слов. Сложность определяется
```
Comment (rss-maksim) [inline src/app/textbook/components/textbook-content/textbook-content.component.html:9]: @Suggestion
Maybe 'разделены' is better

File: src/app/textbook/components/textbook-content/textbook-content.component.scss [hunk 6]
```diff
@@ -46,6 +51,11 @@
   justify-content: space-between;
 }
 
+.tab-link {
+  color: royalblue;
```
Comment (rss-maksim) [inline src/app/textbook/components/textbook-content/textbook-content.component.scss:55]: If it is a common color you can add css variable

File: src/app/textbook/components/textbook-menu/textbook-menu.component.scss [hunk 7]
```diff
@@ -42,4 +42,5 @@ br {
 
 .textbook__menu-btn {
   background-color: var(--color-primary);
+  color: white;
```
Comment (rss-maksim) [inline src/app/textbook/components/textbook-menu/textbook-menu.component.scss:45]: instead of white  you can use variable ```color: var(--color-light)```

File: src/app/textbook/components/word-card/word-card.component.ts [hunk 8]
```diff
@@ -22,8 +22,13 @@ export class WordCardComponent {
 
   constructor(private gameService: MiniGamesHttpService, private store: Store<AppState>, public api: ApiService) {}
 
-  wordSound(audio: string) {
-    playRawSound(`${this.api.githubAssetUrl}/${audio}`);
+  wordSound(arr: any) {
+    let audioArr: string[] = [];
```
Comment (rss-maksim) [inline src/app/textbook/components/word-card/word-card.component.ts:26]: It's a good place where you could use ```map```:

```
return arr.map((str: string) => `${this.api.githubAssetUrl}/${str}`)
```

**Review by rss-maksim — APPROVED** [review]

File: src/app/core/components/header/header.component.scss (hunk 1 above)
Comment (DubichTonya) [reply src/app/core/components/header/header.component.scss:87]: Yes, it doesn't work.

File: src/app/mini-games/sprint/utils/utils.ts (hunk 3 above)
Comment (DubichTonya) [reply src/app/mini-games/sprint/utils/utils.ts:60]: Done

File: src/app/textbook/components/textbook-content/textbook-content.component.html (hunk 5 above)
Comment (DubichTonya) [reply src/app/textbook/components/textbook-content/textbook-content.component.html:9]: No! .. Разделы в учебнике разделены в зависимости от сложности слов.

File: src/app/textbook/components/textbook-content/textbook-content.component.scss (hunk 6 above)
Comment (DubichTonya) [reply src/app/textbook/components/textbook-content/textbook-content.component.scss:55]: Done.

File: src/app/mini-games/sprint/utils/utils.ts (hunk 2 above)
Comment (DubichTonya) [reply src/app/mini-games/sprint/utils/utils.ts:43]: Done.

File: src/app/textbook/components/textbook-content/const.ts (hunk 4 above)
Comment (DubichTonya) [reply src/app/textbook/components/textbook-content/const.ts:25]: Done.

File: src/app/textbook/components/textbook-menu/textbook-menu.component.scss (hunk 7 above)
Comment (DubichTonya) [reply src/app/textbook/components/textbook-menu/textbook-menu.component.scss:45]: Done.

File: src/app/textbook/components/word-card/word-card.component.ts (hunk 8 above)
Comment (DubichTonya) [reply src/app/textbook/components/word-card/word-card.component.ts:26]: Done.
